src/controllers/cron/update_local_smartapi.js

```diff
@@ -173,7 +173,7 @@ const updateSmartAPISpecs = async () => {
     });
 }
 module.exports = () => {
-    cron.schedule('*/10 * * * *', async () => {
+    cron.schedule('* * * * *', async () => {
```

Comment:
@tokebe this now runs every minute?

Reply:
Ah, I made that change during testing and forgot to remove it. Commit fixing this incoming.